src/index.js

```diff
@@ -49,10 +50,23 @@ function splitGuide({
     return pify(fs.readFile)(file, 'utf8')
       .then(contents => ({file, contents}))
   }
+  
+  function promiseChunker(remainingArrayElements, chunkSize) {
```

Comment:
Thanks! It took me a bit to come up with what's here, but I'm pleased with how it turned out.